pravega_provisioner/model/provisioning_logic.py

```diff
@@ -6,7 +6,7 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-from model.constants import Constants
+from constants import Constants
```

Comment:
This does not work, please revert this change:
```
λ python cluster_provisioner.py
Traceback (most recent call last):
  File "cluster_provisioner.py", line 13, in <module>
    from model.provisioning_logic import *
  File "xxx\pravega-tools\pravega_provisioner\model\provisioning_logic.py", line 9, in <module>
    from constants import Constants
ModuleNotFoundError: No module named 'constants'
```

Reply:
I need to figure out what to do, when using Pravega-provisioner as module from scaling script it is not able to find constants.

Reply:
@gaddamas please let me know when you fix this issue to try this PR again.

Reply:
Yes, I fixed it. reverted change to import statement.
Please see files changed and see the diff.